### ISSUE TITLE
How can I use the Model

### ISSUE THREAD

**Issue by Body123** [Retinal-Research/OTE-GAN#2]
is there a link to the pre-trained weights and how I can use the repository

**Comment by ChongQingNoSubway** [comment]
All our model training code was in /OTE-GAN, Without pre-trained weights 
There are two experiments on unsupervised and degradation dataset settings. 

Unsupervised setting prepared the dataset from  Quality label [https://github.com/HzFu/EyeQ] EYEPACS [https://www.kaggle.com/c/diabetic-retinopathy-detection/data] 
->  Change the root dictionary in train.py -> parser.add_argument("--root", default="dataset/preprocessed/ALLDATA", type=str)
parser.add_argument("--file_dir",default="dataset/Label_EyeQ_train.csv", type=str)
->  Run /OTE-GAN/train.py         (Other baseline models followed the same steps)

Degradation setting processing from https://github.com/joanshen0508/Fundus-correction-cofe-Net
-> Change the root dictionary in the train_full.py  parser.add_argument("--root", default="dataset/degratation/pre", type=str)
parser.add_argument("--file_dir",default="dataset/train.csv", type=str)
->  Run /OTE-GAN/train_full.py         (Other baseline models followed the same steps)

**Comment by Body123** [comment]
I didn't understand I will change the  -> Change the root dictionary in the train_full.py parser.add_argument("--root", default="dataset/degratation/pre", type=str)
 to what ?
the second question you save the checkpoint at the end of  every epoch is their criteria to choose the best checkpoint

**Comment by Body123** [comment]
I don't know how to generate mask?
